src/styles/blocks/search-bar--small.css

```diff
@@ -0,0 +1,14 @@
+.search-bar--small {
+  height: 42px;
```

Comment:
Avoid fixing the height of elements if there is no specific requirement for it. Let the content size dictate the container's size to prevent overflow or accidental scrollbars.